python/server/server.sip.in

```diff
@@ -16,37 +16,57 @@ ${DEFAULTDOCSTRINGSIGNATURE}
 
 
 %VirtualErrorHandler serverapi_badrequest_exception_handler
-    PyObject *type, *exception, *traceback, *pyStrException;
+    PyObject *type, *exception, *traceback;
 
     PyErr_Fetch(&type, &exception, &traceback);
-    pyStrException = PyObject_Str(exception);
-    Py_DECREF(pyStrException);
+    // check whether the object is already a unicode string
+    QString pyStrException;
```

Comment:
the 'py' prefix name of the variable is confusing and would imply a PyObject* rather than a QString

Reply:
See https://github.com/qgis/QGIS/pull/59924